### PR TITLE
feature/SGM-15 Artist Sales Module

### PR DESCRIPTION
**Pull request by DylanYoan**
- Add axios endpoints for the backend request
- Add page for the artist sales render

### CODE REVIEW

File: src/pages/Artist/Sales.vue [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    mounted() {
+      $q = useQuasar();
+    }
+  };
+  </script>
+  
+  <style>
+  /* Estilos personalizados si es necesario */
+  </style>
+
```
Comment (temolzinmedtrainer) [inline src/pages/Artist/Sales.vue]: Final de la linea

File: src/store/artist/artistSales/actions.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { api } from "boot/axios";
+
+export const getArtistSales = async ({ commit }) => {
+  await api.get("/api/artist-sales").then((response) => {
+    commit("setArtistSales", response.data.sales);
+  });
+};
```
Comment (temolzinmedtrainer) [inline src/store/artist/artistSales/actions.js]: LINEA

File: src/store/artist/artistSales/getters.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export function stateArtistSales(state) {
+    return state.artistSales;
+  }
```
Comment (temolzinmedtrainer) [inline src/store/artist/artistSales/getters.js]: Linea

File: src/store/artist/artistSales/mutations.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export const setArtistSales = (state, artistSales) => {
+    state.artistSales = artistSales;
+  };
+
```
Comment (temolzinmedtrainer) [inline src/store/artist/artistSales/mutations.js]: Linea

File: src/store/artist/artistSales/state.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export default function () {
+    return {
+      artistSales: {},
+    };
+  }
```
Comment (temolzinmedtrainer) [inline src/store/artist/artistSales/state.js]: Linea

File: src/store/artist/artistSales/actions.js (hunk 2 above)
Comment (DylanYoan) [reply src/store/artist/artistSales/actions.js]: mi maldición

**Review by temolzinmedtrainer — APPROVED** [review]

**Review by temolzin — APPROVED** [review]

**Pull request merged**